contracts/OgioParent.sol

```diff
@@ -29,26 +33,28 @@ abstract contract OgioService is AccessControl {
         Closed
     }
 
+    enum ContributionFrequency {
+        Weekly,
+        Bi_weely,
+        Monthly
+    }
+
     // Define the ROSCAGroup struct
     struct ROSCAGroup {
+        string groupId;
         string groupName;
-        string description;
-        uint256 contributionAmount;
-        uint256 contributionFrequency;
```

Comment:
`contributionAmount` and `contributionFrequency` is a concern of the Escrow, not the group.